framework/lib/components/text-field/formatted-number-input.tsx

```diff
@@ -75,28 +76,41 @@ export const FormattedNumberInput = ({
   preset = 'eu',
   isPercentage = false,
   onChange = identity,
-  value,
+  value: valueProp,
   numberOfDecimals = 2,
   max = Infinity,
   min = -Infinity,
   inputLeftElement,
   inputRightElement,
+  onBlur,
   ...rest
 }: FormattedNumberInputProps) => {
+  const [ v, setV ] = useState(valueProp)
+  const isControlled = typeof valueProp !== 'undefined'
+  const value = isControlled ? valueProp : v
```

Comment:
Okay so I'm very nitpicky here and I know the internal state was called "v" before but could you rename it to valueState? So the distinction between valueState and valueProp is clear

Reply:
Done!